docs/Running-Tachyon-on-a-Cluster.md

```diff
@@ -10,45 +10,50 @@ priority: 1
 
 First download the `Tachyon` tar file, and extract it.
 
-    $ wget https://github.com/amplab/tachyon/releases/download/v{{site.TACHYON_RELEASED_VERSION}}/tachyon-{{site.TACHYON_RELEASED_VERSION}}-bin.tar.gz
```

Comment:
FYI, This file does not seem to have been formatted with the 100 linewrap. (I have tried to approximate that manually).